src/main/java/org/example/service/impl/TraineeServiceImpl.java

```diff
@@ -116,15 +123,58 @@ public void deleteTrainee(DeleteTraineeRequestDto deleteTraineeRequestDto) {
                 LOGGER.warn("Invalid request...");
                 return;
             }
+            List<Trainer> assignedTrainers = traineeDao.getTrainersAssignedToTrainee(deleteTraineeRequestDto.getUsername());
+            for (Trainer trainer : assignedTrainers) {
+                for (Training training : trainer.getTrainings()) {
+                    if (training.getTrainee() != null &&
+                            training.getTrainee().getUsername().equals(deleteTraineeRequestDto.getUsername()) &&
+                            training.getTrainingDate().isAfter(LocalDateTime.now())) {
+
+
+                        GetWorkload getWorkload = new GetWorkload();
```

Comment:
getWorkload is not the best name for this variable according to clean code. Probably noun should be used